v3/mathjax3-ts/core/FindMath.ts

```diff
@@ -0,0 +1,88 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2017 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * @fileoverview  Interfaces and abstract classes for FindMath objects
+ *
+ * @author dpvc@mathjax.org (Davide Cervone)
+ */
+
+import {userOptions, defaultOptions, OptionList} from '../util/Options.js';
+import {ProtoItem} from './MathItem.js';
+
+/*****************************************************************/
+/*
+ *  The FindMath interface
+ */
+
+export interface FindMath {
+    /*
+     * One of two possibilities:  Look through a DOM element,
+     *   or look through an array of strings for delimited math.
+     */
+    findMath(node: Element): ProtoItem[];
+    findMath(strings: string[]): ProtoItem[];
+}
+
+/*****************************************************************/
+/*
+ *  The FindMath class interface
+ */
+export interface FindMathClass {
+    /*
+     * The default options
+     */
+    OPTIONS: OptionList;
```

Comment:
It is.  The `FindMathClass` interface describes the FindMath class object, so `OPTIONS` here means that the abstract class as `OPTIONS` as a static variable, as is indicated in [line 63](https://github.com/mathjax/mathjax-v3/pull/20/files/2e76d08cf358b652cc143f3edd9bde9f8027ca4b#diff-49dd24eef8a13d63cfad88e0117f3d2dR63) below.